src/components/content/Home/Home.js

```diff
@@ -11,12 +11,12 @@ function Home() {
   } = useMatch();
 
   return (
-    <div className="main">
+    <div className='main ml-7 mr-7 sm:ml-0 md:mr-0'>
```

Comment:
Are you doing mobile first? I think if you are doing mobile-first, you don't need the sm right?